compiler/src/iree/compiler/Dialect/HAL/Target/LLVMLinkerUtils.cpp

```diff
@@ -120,6 +120,48 @@ linkBitcodeObjects(Location loc, llvm::Linker &linker, unsigned linkerFlags,
   return success();
 }
 
+LogicalResult linkPathBitcodeFiles(Location loc, llvm::Linker &linker,
```

Comment:
This seems like it should go into RoCM target implementation?

Reply:
Makes sense, done!